### DISCUSSION TITLE
bug-v0.7.5-Audios中Transcript点击全部展示的时候菜单列表不消失

### DISCUSSION THREAD

**Discussion by border** [ZuodaoTech/everyone-can-use-english#1241] [General]
bug-v0.7.5-Audios中Transcript点击全部展示的时候菜单列表不消失
系统: Mac, 版本: v0.7.5

在 点击 "Read Through" 和 “Regenerate” 的时候菜单列表不消失。
<img width="553" alt="image" src="https://github.com/user-attachments/assets/57325e53-944b-4c45-ab61-48b6aef42060">

<img width="1002" alt="image" src="https://github.com/user-attachments/assets/0bc037f4-a7ef-48c9-b96f-b39ef9d703a7">

**Comment by an-lee** [comment]
v0.7.6 版本已经修复，请更新到最新版即可。

**Comment by border** [comment]
貌似检查更新也有些问题，我在v0.7.5版本点击查看最新版本，没有提示有新的版本。

**Reply by an-lee** [reply]
嗯，我再检查下。
